content/tormobile/tormobile-6/contents.lr

```diff
@@ -0,0 +1,11 @@
+_model: question
+---
+title: Do I need Tor Browser for Android and Orbot, or only one?
+---
+seo_slug: tor-browser-for-android-and-orbot
+---
+description:
+While both Tor Browser for Android and Orbot are great, they serve different purposes, so unless you need both of their functionalities, then you don't need both.
```

Comment:
IMHO, this could be explained a bit better, 

> so unless you need both of their functionalities, then you don't need both.

seems redundant to me and could be omitted maybe.

Reply:
I agree, I'll change that